src/planscape/conditions/urls.py

```diff
@@ -2,5 +2,6 @@
 from . import views
 
 urlpatterns = [
-    path('wms/', views.wms)
+    path('wms/', views.wms),
+    path('regions/', views.regions)
```

Comment:
something like config/ might be more accurate?

Reply:
Done!